config/routes.rb

```diff
@@ -17,13 +17,16 @@
       put 'picture', to: 'people#update_picture'
       get 'picture'
       get 'fws', to: 'people#export_fws'
+      resources :skills, controller: 'people/skills'
     end
 
+    resources :people_skills, controller: 'people/skills'
```

Comment:
Diese resource braucht es glaub nicht, wurde ja bereits in der people resource definiert

Reply:
Doch die wird zumindest beim New genutzt (bei Edit glaube ich auch). Das liegt daran dass Ember bei peopleSkillRecord.save() einfach ein Post/Patch an /api/people_skills sendet...

Reply:
GIbt es da keine Möglichkeit den Ember rest service umzuschreiben damit er nur noch eine Resource hat? Ist schon nicht so toll wenn nur wegen Ember zwei URL's auf den selben Controller zeigen